<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>瓦力音频播放器</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="3" column="0" colspan="3">
     <widget class="QFrame" name="frame">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QPushButton" name="previous">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="toolTip">
          <string>上一首</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/images/uprevious.png</normaloff>:/images/uprevious.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>31</width>
           <height>31</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="play">
         <property name="minimumSize">
          <size>
           <width>37</width>
           <height>37</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>37</width>
           <height>37</height>
          </size>
         </property>
         <property name="toolTip">
          <string>播放</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/images/uplay.png</normaloff>:/images/uplay.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>37</width>
           <height>37</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="next">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="toolTip">
          <string>下一首</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/images/unext.png</normaloff>:/images/unext.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>31</width>
           <height>31</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="lbCurTime">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string> 00:00</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QSlider" name="slTime">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QLabel" name="lbTotalTime">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>00:00 </string>
         </property>
        </widget>
       </item>
       <item row="0" column="6">
        <widget class="QPushButton" name="icon_Volume">
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="toolTip">
          <string>音量</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/images/volume2.png</normaloff>:/images/volume2.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="8">
        <widget class="QPushButton" name="mode_change">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="toolTip">
          <string>列表循环</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/images/uloop1.png</normaloff>:/images/uloop1.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="7">
        <widget class="QSlider" name="slVolume">
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1" rowspan="2" colspan="2">
     <widget class="QWidget" name="widget" native="true">
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <property name="spacing">
        <number>3</number>
       </property>
       <item row="1" column="0">
        <widget class="QTextEdit" name="textEdit"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="CurvePlot" name="ploter" native="true"/>
    </item>
    <item row="0" column="0" rowspan="3">
     <widget class="QWidget" name="widget_2" native="true">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <property name="spacing">
        <number>3</number>
       </property>
       <item row="0" column="2">
        <widget class="QPushButton" name="menu_but">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="toolTip">
          <string>返回</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/images/menu.png</normaloff>:/images/menu.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>19</width>
           <height>19</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="QListWidget" name="lwList">
         <property name="styleSheet">
          <string notr="true">QListWidget::item{height: 40px}

QListWidget {
	background-color: rgb(255, 255, 255);
}</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLineEdit" name="find_Edit"/>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="find_but">
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="toolTip">
          <string>搜索</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/images/find.png</normaloff>:/images/find.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>19</width>
           <height>19</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QListWidget" name="lwList_f">
         <property name="styleSheet">
          <string notr="true">QListWidget::item{height: 40px}

QListWidget {
	background-color: rgb(255, 255, 255);
}</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdd"/>
   <addaction name="actionClear"/>
   <addaction name="actionMinus"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>738</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <action name="actionAdd">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/file.png</normaloff>:/images/file.png</iconset>
   </property>
   <property name="text">
    <string>添加播放文件(&amp;F)</string>
   </property>
   <property name="toolTip">
    <string>添加播放文件</string>
   </property>
  </action>
  <action name="actionList">
   <property name="icon">
    <iconset>
     <normaloff>:/images/list.png</normaloff>:/images/list.png</iconset>
   </property>
   <property name="text">
    <string>文件栏显示(&amp;H)</string>
   </property>
   <property name="toolTip">
    <string>文件栏显示</string>
   </property>
  </action>
  <action name="actionMinus">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/minus.png</normaloff>:/images/minus.png</iconset>
   </property>
   <property name="text">
    <string>minus</string>
   </property>
   <property name="toolTip">
    <string>移除歌曲</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/clear.png</normaloff>:/images/clear.png</iconset>
   </property>
   <property name="text">
    <string>清空播放栏</string>
   </property>
   <property name="toolTip">
    <string>清空播放栏</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CurvePlot</class>
   <extends>QWidget</extends>
   <header>CurvePlot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
